docs/guides/getting_started/intro.md

```diff
@@ -99,33 +105,37 @@ the Console.
 
 [!code-csharp[Async Context](samples/intro/logging.cs)]
 
+[API Documentation]: xref:Discord.Rest.BaseDiscordClient#Discord_Rest_BaseDiscordClient_Log
+
 ### Creating a Discord Client
 
 Finally, we can create a connection to Discord. Since we are writing
-a bot, we will be using a [DiscordSocketClient], along with socket
+a bot, we will be using a [DiscordSocketClient] along with socket
 entities. See the [terminology](terminology.md) if you're unsure of
 the differences.
 
 To do so, create an instance of [DiscordSocketClient] in your async
 main, passing in a configuration object only if necessary. For most
 users, the default will work fine.
 
-Before connecting, we should hook the client's log event to the
+Before connecting, we should hook the client's `Log` event to the
 log handler that was just created. Events in Discord.Net work
 similarly to other events in C#, so hook this event the way that
 you typically would.
 
-Next, you will need to 'login to Discord' with the `LoginAsync` method.
+Next, you will need to "login to Discord" with the `LoginAsync` 
+method.
 
 You may create a variable to hold your bot's token (this can be found
 on your bot's application page on the [Discord Applications Portal]).
+
 ![Token](images/intro-token.png)
 
 >[!IMPORTANT]
 Your bot's token can be used to gain total access to your bot, so
-**do __NOT__ share this token with anyone!** It may behoove you to
-store this token in an external file if you plan on distributing the
-source code for your bot.
+**do __NOT__ share this token with anyone else!** It may behoove you 
```

Comment:
I think if we are going to stress that this is something really important, we should use straightforward wording. Swap "behoove" with "It is responsible to store this token in...", or other wording.

Reply:
Won't fix. Main contribs preferred the word "behoove" to stay.